src/urh/dev/PCAPNG.py

```diff
@@ -0,0 +1,114 @@
+import os
+import struct
+import time
+import math
+
+from urh.util.Logger import logger
+
+from urh.signalprocessing.Message import Message
+
+
+class PCAPNG(object):
+    def __init__(self):
+        self.__messages = []
+        self.__messages_timestamps = []
+        self.__filename = ""
+
+    def __build_pcapng_shb(self, shb_userappl : str = "", shb_hardware : str = "") -> bytes:
+        BLOCKTYPE = 0x0A0D0D0A
+        HEADERS_BLOCK_LENGTH = 28
+        MAGIC_NUMBER = 0x1A2B3C4D
+        VERSION_MAJOR, VERSION_MINOR = 1,0
+        SECTIONLENGTH = 0xFFFFFFFFFFFFFFFF # -1 => Not specified
+
+        shb_userappl_padded_len = math.ceil(len(shb_userappl)/4) * 4
+        shb_hardware_padded_len = math.ceil(len(shb_hardware)/4) * 4
+
+        total_block_len = HEADERS_BLOCK_LENGTH
+        if (shb_userappl_padded_len > 0):
```

Comment:
braces in an `if` are not needed (applies also on other locations in this PR):
```suggestion
        if shb_userappl_padded_len > 0:
```

Reply:
Changed all occurences in PR